.github/workflows/build_image.yml

```diff
@@ -14,6 +14,9 @@ on:
       tag_name:
         type: string
         default: latest
+      extra-build-args:
+        type: string
+        default: ""
```

Comment:
future proofing I hope!

Reply:
When the MTJNGO container image is built in CircleCi, some build arguments are passed to docker build.  This enables us to do the same thing in Github Actions.
```
-    steps:
-      - git-shallow-clone/checkout
-      - aws-ecr/build-and-push-image:
-          account-url: AWS_ECR_ACCOUNT_URL_V2
-          aws-access-key-id: AWS_ECR_ACCESS_KEY_ID_V2
-          aws-secret-access-key: AWS_ECR_SECRET_ACCESS_KEY_V2
-          checkout: false
-          dockerfile: Dockerfile
-          no-output-timeout: 20m
-          region: AWS_ECR_REGION
-          repo: "${AWS_ECR_RESOURCE_NAME_PREFIX_V2}"
-          skip-when-tags-exist: false
-          tag: "${CIRCLE_SHA1},${CIRCLE_BRANCH}"
-          extra-build-args: "--build-arg APP_CODE=BTMTJNGO --build-arg NPM_TOKEN=${NPM_TOKEN} --build-arg VERSION=${CIRCLE_SHA1:0:7}"
- 
```